build.sbt

```diff
@@ -9,6 +9,7 @@ lazy val freestyle = (crossProject in file("freestyle"))
   .settings(name := "freestyle")
   .jsSettings(sharedJsSettings: _*)
   .settings(libraryDependencies ++= Seq(%("scala-reflect", scalaVersion.value)))
+  .settings(libraryDependencies += "com.47deg" %%% "iota-core" % "0.0.2")
```

Comment:
Yep. I wasn't sure exactly what I needed to do and I wanted to get this PR built and passing ASAP.